antlr/Makefile

```diff
@@ -0,0 +1,30 @@
+# GNU Make >= 4.3
+
+## Configuration
+
+OUTPUT_DIR = parser/grulev3
+
+ANTLR = antlr4
```

Comment:
I've tried to run `make all` with the ANTLR 4.13 jar renamed to `antlr4` (tried `antlr4.jar` as well) and moved to the antlr directory here. I get errors about antlr4 not being found though:

```
jonathan.forscher@MacBookPro antlr % make all
antlr4 -Dlanguage=Go -o 'parser/grulev3' -package 'grulev3' -visitor grulev3.g4
make: antlr4: No such file or directory
make: *** [parser/grulev3/grulev3Lexer.interp] Error 1
```

Building via the instructions in the project's ANTLR.MD project does "work" though it generates in the wrong directory.